src/auth/users.respository.ts

```diff
@@ -37,8 +41,35 @@ export class UsersRepository extends Repository<User> {
       }
     }
   }
+
   async getUser(idUser: string): Promise<User> {
     const user = this.findOne({ where: { id: idUser } });
     return user;
   }
+
+  async updateOne(userId: string, imagePath: string): Promise<User> {
+    try {
+      const query = await this.createQueryBuilder()
+        .update(User)
+        .set({ profileImage: imagePath })
+        .where({ id: userId })
+        .execute();
+      if (query.affected === 1) {
+        const user = await this.getUser(userId);
+        return user;
+      } else {
+        throw new NotFoundException();
+      }
+    } catch (error) {
+      throw new InternalServerErrorException();
+    }
+  }
+
+  async getProfileImage(user: User, res): Promise<Object> {
```

Comment:
Should return <any> type
![return-file](https://user-images.githubusercontent.com/77193627/156460766-83f55926-efe7-4b06-814c-0f56256b3eff.png)
.